### DISCUSSION TITLE
Restore the RSS button

### DISCUSSION THREAD

**Discussion by GreenLunar** [Floorp-Projects/Floorp#570] [Feature Request]
Please restore the RSS button.
https://openrss.org/blog/browsers-should-bring-back-the-rss-button

Please also see arguments listed in this program.
https://openuserjs.org/scripts/sjehuda/Newspaper

Long live RSS ;)
https://camendesign.com/blog/rss_is_dying

**Comment by Nitrousoxide** [comment]
I use 
https://addons.mozilla.org/en-US/firefox/addon/rsspreview/

**Comment by GreenLunar** [comment]
In contrast to addons, including this feature by default would:

1) Increase awareness to RSS, which is more private than web browsing
and email.

2) You would avoid unwillful disclosure of user identity which could be
revealed by server attempt to disclose used set of addons.

> ### [Browser extensions don't cut it](https://openrss.org/blog/browsers-should-bring-back-the-rss-button#browser-extensions-don't-cut-it)
> If you want the built-in RSS button back in your browser, many browser vendors will recommend using a third-party browser extension to replicate the functionality.
> 
> But browser extensions don't help much with RSS adoption. Potential new RSS users or those who may already assume websites just don't have RSS feeds anymore are unlikely to go looking for a browser extension.
> 
> In addition, browser extensions make your [browser less secure](https://www.howtogeek.com/188346/why-browser-extensions-can-be-dangerous-and-how-to-protect-yourself/), invade your privacy, and slow down your browser. And many extensions aren't actively maintained, so they can fall out-of-date and suddenly stop working.

The big web browser vendors have turned RSS into a privacy feature in
addition to what it originally is, an accessibility and publishing mean.

See more arguments for RSS in the "Help" dialog of the "Newspaper"
userscript I referred to (three vertical dots which you can see upon
reading a feed).
